### PR TITLE
Kobo.Kobo version 4.26.14705

### PR DESCRIPTION
**Pull request by mddtest**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/15663)

### CODE REVIEW

File: manifests/k/Kobo/Kobo/4.26/Kobo.Kobo.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Kobo.Kobo
+PackageVersion: 4.26
+PackageLocale: en-US
+Publisher: Kobo
+PackageName: Kobo
+License: Kobo
```
Comment (jedieaston) [inline manifests/k/Kobo/Kobo/4.26/Kobo.Kobo.locale.en-US.yaml:8]:
```suggestion
License: (c) 2021 Rakuten Kobo Inc.

```

File: manifests/k/Kobo/Kobo/4.26/Kobo.Kobo.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Kobo.Kobo
+PackageVersion: 4.26
```
Comment (jedieaston) [inline manifests/k/Kobo/Kobo/4.26/Kobo.Kobo.installer.yaml:4]:
```suggestion
PackageVersion: 4.26.14705

```

(So it'll match the ARP table).

**Review by jedieaston — CHANGES_REQUESTED** [review]

**Review by mddtest — COMMENTED** [review]
updated short description

File: manifests/k/Kobo/Kobo/4.26/Kobo.Kobo.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: Kobo.Kobo
+PackageVersion: 4.26
```
Comment (jedieaston) [inline manifests/k/Kobo/Kobo/4.26/Kobo.Kobo.yaml:4]:
```suggestion
PackageVersion: 4.26.14705

```

**Review by jedieaston — CHANGES_REQUESTED** [review]
The enclosing folder should be 4.26.14705, not 4.26 to match the spec. Once that's fixed, you should be good to go.

**Review by mddtest — COMMENTED** [review]
updated

**Comment by mddtest** [comment]
> 
> 
> @mddtest One last thing: can you move it to the folder `manifests/k/Kobo/Kobo/4.26.14705`? The enclosing folder name has to match the `PackageVersion`, per policy.

I don;t believe I have that access.

**Comment by jedieaston** [comment]
Just rename the folder on your machine from 4.26 to 4.26.14705, and commit all of the changes. GitHub will update your PR with the change.

**Comment by mddtest** [comment]
> 
> 
> Just rename the folder on your machine from 4.26 to 4.26.14705, and commit all of the changes. GitHub will update your PR with the change.
what am I doing wrong

image

**Comment by mddtest** [comment]
I think I did the change correctly @jedieaston

**Comment by jedieaston** [comment]
It doesn't appear that it was pushed to GitHub. Do you have Git installed on your machine?

**Pull request closed without merging**